core/src/main/resources/application.conf

```diff
@@ -52,8 +52,14 @@ system {
   // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
   max-retries = 10
 
-  //If 'true' then when Cromwell starts up, it tries to restart incomplete workflows
+  // If 'true' then when Cromwell starts up, it tries to restart incomplete workflows
   workflow-restart = true
+
+  // Cromwell will launch up to N submitted workflows at a time. That number is tunable here
```

Comment:
@cjllanwarne re your first comment, no that is not the case. This doesn't limit the number of workflows in flight, just the number of workflows it fetches from the store on any one pull.

As an example, at least in the past we've had issues with too many things submitted at once (which e.g. led to bbq in Rawls). This allows a reasonable rate to be determined by Cromwell.